src/components/profile-youtube-playlist/profile-youtube-playlist.js

```diff
@@ -0,0 +1,102 @@
+import { Helmet } from 'react-helmet';
+import React, { Component } from 'react';
+import Carousel from '@brainhubeu/react-carousel';
+import '@brainhubeu/react-carousel/lib/style.css';
+import PropTypes from 'prop-types';
+
+export default class YoutubePlaylist extends Component {
```

Comment:
why not use hooks?

Reply:
updated the component to use react hooks